packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2437,9 +2440,18 @@ export class ContainerRuntime
 					throw error2;
 				}
 			} else {
-				// pre-0.58 error message: orderSequentiallyCallbackException
-				this.closeFn(new GenericError("orderSequentially callback exception", error));
+				this.closeFn(
+					wrapError(
```

Comment:
This will result in a `GenericError` even if the inner error is of a known type like `DataProcessingError`.  You might want to use `normalizeError` instead which will return the same object back if it's already got an `errorType` and looks like one of our errors.

Note that `assert`s and bare `Error` or `LoggingError` will get wrapped by `normalizeError`.

Reply:
@andre4i - Curious why you decided to keep `wrapError` instead of switching to `normalizeError`

Reply:
@markfields I went with your other suggestion of changing the generic error's error message to include the original error vs keeping the original error. For this context I thought it would be a cheaper API usage considering I need an extra telemetry property.